FakebookNotifications/FakebookNotifications.WebApi/FakebookNotifications.WebApi.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
+    <UserSecretsId>c5cf906f-d5db-4953-9447-58d487547d5b</UserSecretsId>
```

Comment:
this is fine, but i'm curious what you're using user secrets for here

Reply:
nothing was added, I clicked on "Manage user secrets" to see what was there and it just created that.